addOns/encoder/src/main/java/org/zaproxy/addon/encoder/processors/predefined/Base64Encoder.java

```diff
@@ -21,12 +21,26 @@
 
 import java.io.IOException;
 import java.util.Base64;
+import org.parosproxy.paros.control.Control;
+import org.zaproxy.addon.encoder.EncodeDecodeParam;
+import org.zaproxy.addon.encoder.ExtensionEncoder;
 
 public class Base64Encoder extends DefaultEncodeDecodeProcessor {
 
     @Override
     protected String processInternal(String value) throws IOException {
-        // Returns based on StandardCharsets.ISO_8859_1
-        return Base64.getEncoder().encodeToString(value.getBytes());
+        EncodeDecodeParam encDecParam =
+                Control.getSingleton()
+                        .getExtensionLoader()
+                        .getExtension(ExtensionEncoder.class)
+                        .getParams();
+        if (encDecParam.isBase64DoBreakLines()) {
+            return new String(
+                    Base64.getMimeEncoder().encode(value.getBytes(encDecParam.getBase64Charset())),
```

Comment:
Per: https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html this wraps at 76, which is the same as: https://github.com/zaproxy/zaproxy/blob/baa415b72755cc5348de8a0259e780152d8e6345/zap/src/main/java/org/parosproxy/paros/extension/encoder/Base64.java#L212-L213

Reply:
Could use: https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html#getMimeEncoder-int-byte:A- if for some reason `\n` is preferred vs `\r\n`.